frontend/src/components/Forms/AddPermitAmendmentForm.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import RenderDate from "@/components/common/RenderDate";
+import RenderAutoSizeField from "@/components/common/RenderAutoSizeField";
+import { Form, Button, Col, Row, Popconfirm, Tabs } from "antd";
+import * as FORM from "@/constants/forms";
+import { required, maxLength, dateNotInFuture } from "@/utils/Validate";
+import { resetForm } from "@/utils/helpers";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+  submitting: PropTypes.bool.isRequired,
+};
+
+const { TabPane } = Tabs;
+
+export const AddPermitAmendmentForm = (props) => (
+  <Form layout="vertical" onSubmit={props.handleSubmit}>
+    <Tabs defaultActiveKey="1" style={{ marginTop: "-30px" }}>
+      <TabPane tab="Info" key="1">
+        <br />
```

Comment:
Are we using hard line breaks on our other forms? Styling with our HTML seems like a tough pattern to manage since it blurs the line between HTML/CSS

Reply:
I actually removed this before seeing your comment, as the tabs were placeholder/wip for upload, which won't end up using tabs anyways.